### DISCUSSION TITLE
SmartCrates based on directory structure

### DISCUSSION THREAD

**Discussion by ghost** [rshea0/enjinn#54] [Ideas]
Hi Ryan, 
Just found this great app, thanks for the great job and the effort. Thinking about register the app, but I have a question about SmartCrates.
It's possible to add SmartCrates based on directory structure? 
Ex. I have my music organized like this

Year 
----- Genere
--------- SubGenere

2021
----- Rock
---------- Progresive

I look at the tuneup-example.yaml but I don't find any clue if this is possible.

If not, I think maybe it's a good feature to implement. Ask for a root folder and start creating crates following directory tree.

If I can help u in some way about this, don't heristate to ask me.
Daniel

**Comment by shayded-exe** [comment]
Not something currently supported. I'll consider adding this, but there's a lot that's higher priority.

I'm also currently on a break from development, but I'll be back to providing updates soon.

**Comment by ghost** [comment]
Hi Ryan, don't worry, was just an idea.
Hope u 're well and enjoing family, friends and your spare time.
